Tutorials/Level1Tutorial.md

```diff
@@ -0,0 +1,780 @@
+# Level 1 Tutorial
+
+## Table of Contents
+1. [What is SST](#WhatIsSST)
+2. [Components of SST](#ComponentsOfSST)
+3. [Basic Installation](#BasicInstallation)
+4. [Basic Commands](#BasicCommands)
+5. [Basic Runtime Functions](#BasicRuntimeFunctions)
+6. [Collecting Data from SST](#CollectingData)
+7. [Links to Documentation](#LinksToDocumentation)
+
+## What is SST: <a name="WhatIsSST"></a>
+  To combat the ever-changing and multifaceted technologies of the HPC 
+  community, an effort was made to create a unified tool box used to evaluate 
+  these challenges. The Structural Simulation Toolkit (SST) an open, modular, 
+  parallel, multi-criteria, multi-scale simulation framework  was created to 
```

Comment:
I don't know what "multi-criteria" refers to.

Reply:
resolved in a48b6c9